### PR TITLE
add all new ores with receiptes, but without generation in the world

### LINKED ISSUE #25: Create gem ore blocks and gem items (check word file, 28 ore varieties)

### PR DESCRIPTION
**Pull request by koliagas**

### CODE REVIEW

File: src/main/resources/assets/russianmagic/models/item/amethyst_block_ore.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "parent": "russianmagic:block/chrysolite_block_ore"
```
Comment (Strannick112) [inline src/main/resources/assets/russianmagic/models/item/amethyst_block_ore.json]: Perhaps an amethyst_block_ore?

File: src/main/resources/assets/russianmagic/lang/ru_ru.json [hunk 2]
```diff
@@ -43,9 +43,100 @@
...
 
   "block.russianmagic.aquamarine_block": "Блок аквамарина",
-  "block.russianmagic.aquamarine_block_ore": "Руда аквамарина"
+  "block.russianmagic.aquamarine_block_ore": "Руда аквамарина",
 
   "block.russianmagic.rhinestone_block": "Блок горного хрусталя",
-  "block.russianmagic.rhinestone_block_ore": "Руда горного хрусталя"
+  "block.russianmagic.rhinestone_block_ore": "Руда горного хрусталя",
 
+  "block.russianmagic.jasper": "Яшма>",
```
Comment (Strannick112) [inline src/main/resources/assets/russianmagic/lang/ru_ru.json]: Яшма>

File: src/main/resources/assets/russianmagic/lang/ru_ru.json [hunk 3]
```diff
@@ -43,9 +43,100 @@
...
+  "block.russianmagic.jasper_block": "Блок яшмы>",
```
Comment (Strannick112) [inline src/main/resources/assets/russianmagic/lang/ru_ru.json]: Блок яшмы>

File: src/main/resources/assets/russianmagic/lang/ru_ru.json [hunk 4]
```diff
@@ -43,9 +43,100 @@
...
+  "block.russianmagic.jasper_block_ore": "Руда яшмы",
+
+  "block.russianmagic.chrysolite": "Xризолит",
+  "block.russianmagic.chrysolite_block": "Блок хризолита",
+  "block.russianmagic.chrysolite_block_ore": "Руда хризолита",
+
+  "block.russianmagic.garnet": "Uранат",
```
Comment (Strannick112) [inline src/main/resources/assets/russianmagic/lang/ru_ru.json]: Uранат

**Review by Strannick112 — APPROVED** [review]

**Review by Strannick112 — APPROVED** [review]

**Review by Regela — APPROVED** [review]

**Pull request merged**